async-include.js

```diff
@@ -1,17 +1,21 @@
 /* async-inlude - by Peter Coles http://mrcoles.com */
-(function($) {
-    $(function() {
-        $('[data-async-url]').each(function() {
-            var $this = $(this),
-                url = $this.data('async-url');
-            $.ajax({
-                url: url,
-                dataType: 'html',
-                type: 'get',
-                success: function(html) {
-                    $this.replaceWith(html);
-                }
-            });
-        });
+(function ($) {
+  $.fn.asyncInclude = function() {
+    return this.each(function() {
+      var that = $(this);
```

Comment:
I like to name jQuery selections starting with a dollar sign. Any reason you changed it from `$this` to `that`? The `context` param is nice though.